src/main/java/ch/saunah/saunahbackend/service/PriceService.java

```diff
@@ -0,0 +1,90 @@
+package ch.saunah.saunahbackend.service;
+
+import ch.saunah.saunahbackend.dto.PriceBody;
+import ch.saunah.saunahbackend.model.Price;
+import ch.saunah.saunahbackend.repository.PriceRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.webjars.NotFoundException;
+
+import java.util.Objects;
+
+/**
+ * This class contains add, edit, get and remove methods for price structures
+ */
+@Service
+public class PriceService {
+
+    @Autowired
+    private PriceRepository priceRepository;
+
+    /**
+     * Adds a new Price structure to the Database
+     * @param priceBody the parameter needed to create a Price structure
+     * @return a new Price structure
+     * @throws NullPointerException if a required Object is null
+     */
+    public Price addPrice(PriceBody priceBody) throws NullPointerException {
+        Objects.requireNonNull(priceBody, "PriceBody must not be null!");
+        Price price = new Price();
+        setPriceFields(price, priceBody);
+
+        return priceRepository.save(price);
+    }
+
+    /**
+     * Lets the user edit an existing price structure
+     * @param id The id of the Price structure that shall be edited
+     * @param priceBody the parameters of the price structure that is being edited
+     * @return The price structure with the changed values
+     * @throws NullPointerException
+     */
+    public Price editPrice(int id, PriceBody priceBody) throws NullPointerException {
+        Price editPrice = getPrice(id);
+        setPriceFields(editPrice, priceBody);
+        return priceRepository.save(editPrice);
+    }
+
+    /**
+     * Remove an existing price structure from the database
+     * @param id the id of the price structure that should be deleted
+     * @return message if price structure has been successfully been deleted
+     * @throws NullPointerException
+     */
+    public String removePrice(int id) throws NullPointerException {
+        priceRepository.deleteById(id);
```

Comment:
check if entity exists in database before deleting, same as removeSauna in saunaService

Reply:
Done